packages/typespec-vscode/.vscodeignore

```diff
@@ -6,12 +6,12 @@
 
 # Then explicitly include what we want to ship in VSIX
 !dist/**/*.tmLanguage
-!dist/**/*.js
-!dist/**/*.cjs
-!dist/**/*.js.map
+!dist/src/**/*.js
+!dist/src/**/*.cjs
+!dist/src/**/*.js.map
+!dist/swagger-ui*.js
```

Comment:
i think this will exclude swagger-ui from vsix package and cause problem in preview scenario, won't it?

Reply:
Those undo the global exclusion on top. This is act as an include file

Reply:
The change here was to prevent the tests to be included in the bundle